lib/services/start-service.ts

```diff
@@ -0,0 +1,115 @@
+import { ChildProcess } from "child_process";
+import { IChildProcess } from "../common/declarations";
+import {
+	IKeyCommandHelper,
+	IValidKeyCommands,
+} from "../common/definitions/key-commands";
+import { injector } from "../common/yok";
+import { IProjectData } from "../definitions/project";
+import { IStartService } from "./../definitions/start-service.d";
+
+export default class StartService implements IStartService {
+	ios: ChildProcess;
+	android: ChildProcess;
+	verbose: boolean = false;
+
+	constructor(
+		private $keyCommandHelper: IKeyCommandHelper,
+		private $childProcess: IChildProcess,
+		private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants,
+		private $projectData: IProjectData,
+		private $logger: ILogger
+	) {}
+
+	toggleVerbose(): void {
+		this.verbose = true;
+		this.$logger.info(
+			this.verbose ? `Verbose logging enabled` : `Verbose logging disabled`
+		);
+	}
+
+	format(data: Buffer, platform: string) {
+		return data;
+	}
+
+	async runForPlatform(platform: string) {
+		const platformLowerCase = platform.toLowerCase();
+		(this as any)[platformLowerCase] = this.$childProcess.spawn(
+			"../nativescript-cli/bin/ns",
```

Comment:
This seems to always resolve relative to the project folder.  should be just `ns`?